tracer/dependabot/Datadog.Dependabot.Integrations.csproj

```diff
@@ -237,8 +237,8 @@
 
     <!-- Integration: Microsoft.VisualStudio.TestPlatform.MSTest.TestAdapter -->
     <!--    Assembly: Microsoft.VisualStudio.TestPlatform.MSTest.TestAdapter -->
-    <!-- Latest package https://www.nuget.org/packages/MSTest.TestAdapter/3.9.3 -->
-    <PackageReference Include="MSTest.TestAdapter" Version="3.9.3" />
+    <!-- Latest package https://www.nuget.org/packages/MSTest.TestAdapter/3.10.0 -->
+    <PackageReference Include="MSTest.TestAdapter" Version="3.10.0" />
```

Comment:
Appears to have broken the tests for this one

Reply:
Fixed in #7343